src/devices/HCSR04/samples/README.md

```diff
@@ -0,0 +1,4 @@
+﻿# TODO: This needs to be determined
+
+Help Wanted Please
```

Comment:
Do you have specific questions regarding the content that should go here? Here is a very good example of one https://github.com/dotnet/iot/blob/master/src/devices/Mcp3008/samples/README.md. Note that all of our current README's for samples have fritz diagrams. The main idea of this file, should be that if a new developer had an HCSR04 and a RaspberryPi and wanted to try it out but didn't know how to even connect it, your README should be good enough to get him into the right path of connecting it and running the first hello world (or in your case, your sample) on it. I am happy to help you out with any questions you have about the contents of the file.

Reply:
I will fix a simple Fritz diagram, the text in the readme here is just from the boilerplate template.